### DISCUSSION TITLE
Help needed for the following error - Could not resolve to a node with the global id

### DISCUSSION THREAD

**Discussion by shivajipotnuru** [octokit/octokit.net#2621] [Q&A]
I'm trying to assign a pull request reviewer to a Pull request

I'm getting the following error.
`{"message":"Validation Failed","errors":["Could not resolve to a node with the global id of 'MDQ6VGVhbTY4MA=='."],"documentation_url":"https://docs.github.com/enterprise-server@3.6/rest/reference/pulls#request-reviewers-for-a-pull-request"}`

I'm trying to authenticate the service using 

> 'GITHUB APP`
I've given Read & write permissions to the app.

here is my code.
`
           
            var teams = new List<string> { "chatbot-reviewers" };

            var requestTeams = PullRequestReviewRequest.ForTeamReviewers(teams);
            try
            {
                await client.PullRequest.ReviewRequest.Create("EIA", repositoryName, prNumber, requestTeams);
                return new OkObjectResult("Pull Request Assigned to the Chatbot Reviewers Team.");

            }
            catch (Exception e)
            {

                logger.LogInformation(e.Message);
                return new BadRequestResult();
            }
`

**Comment by kfcampbell** [comment]
Hmm....looking at the error message, `MDQ6VGVhbTY4MA==` base64-decodes to `04:Team680`. That and the talk of node IDs in the error make me think this would be GraphQL-related, but the docs URL provided and the code linked are all REST-related. 

Are you able to make other requests with the same authentication?

**Reply by shivajipotnuru** [reply]
Hello, keegan, 

In my code I'm using the same authentication client , I was able to create Issue & Releases as well, but however when i try to assign the pull request to a specific team I'm getting the following error.

Any help on this is really appreciated!

**Reply by shivajipotnuru** [reply]
Here is the one more function with the same client, i was able to create the release package.
```
 private static async Task<IActionResult> CreateRelasePackage(string repositoryName, string tagName, string releaseTitle, string preRelease, GitHubClient client, ILogger logger)
        {
            var releaseMetadata = new NewRelease(tagName);
            releaseMetadata.GenerateReleaseNotes = true;
            releaseMetadata.Name = releaseTitle;
            releaseMetadata.Prerelease = bool.Parse(preRelease); 
            try
            {
                var newRelease = await client.Repository.Release.Create("EIA", repositoryName, releaseMetadata);
                logger.LogInformation("Github Package Created Successfully!");
                return await GenerateReleaseNotes(repositoryName, client, logger);
            }
            catch (Exception e)
            {
                logger.LogInformation(e.Message);
                return new BadRequestResult();
            }

        }
```
